lib/dash/dash_parser.js

```diff
@@ -957,11 +957,13 @@ shaka.dash.DashParser.prototype.parseAdaptationSet_ = function(context, elem) {
   var language =
       shaka.util.LanguageUtils.normalize(elem.getAttribute('lang') || 'und');
 
+  var label = elem.getAttribute('label');
```

Comment:
I was unable to find any mention of this attribute in the DASH spec (2014) or the latest DASH-IF-IOP (v4.0).  I don't object to you adding it, though.

Please add a comment that this is a non-standard attribute supported by Kaltura.